### PR TITLE
fix: swap filenames for book-hold canceling and canceled

### PR DESCRIPTION
**Pull request by dtojikulov**
For some reason `book-hold-canceled.ts` and `book-hold-canceling-failed.ts` events were switched in places, in other words one file was exporting opposing event.

fixed that ;)

### CODE REVIEW

File: libs/lending/domain/src/index.ts [hunk 1]
```diff
@@ -4,8 +4,8 @@ export * from './lib/book/book-on-hold';
 export * from './lib/book/book-duplicate-hold-found.event';
 export * from './lib/events/book-check-out-failed';
 export * from './lib/events/book-checked-out';
-export * from './lib/events/book-hold-canceled';
 export * from './lib/events/book-hold-canceling-failed';
+export * from './lib/events/book-hold-canceled';
```
Comment (MaciejSikorskiRamp) [inline libs/lending/domain/src/index.ts]: I think the previous order was correct

File: libs/lending/domain/src/lib/patron.ts [hunk 2]
```diff
@@ -4,8 +4,8 @@ import { AvailableBook } from './book/available-book';
 import { BookOnHold } from './book/book-on-hold';
 import { BookCheckOutFailed } from './events/book-check-out-failed';
 import { BookCheckedOut } from './events/book-checked-out';
-import { BookHoldCancelingFailed } from './events/book-hold-canceled';
-import { BookHoldCanceled } from './events/book-hold-canceling-failed';
+import { BookHoldCancelingFailed } from './events/book-hold-canceling-failed';
+import { BookHoldCanceled } from './events/book-hold-canceled';
```
Comment (MaciejSikorskiRamp) [inline libs/lending/domain/src/lib/patron.ts]: Same here

**Review by MaciejSikorskiRamp — CHANGES_REQUESTED** [review]

File: libs/lending/domain/src/index.ts (hunk 1 above)
Comment (dtojikulov) [reply libs/lending/domain/src/index.ts]: done

File: libs/lending/domain/src/lib/patron.ts (hunk 2 above)
Comment (dtojikulov) [reply libs/lending/domain/src/lib/patron.ts]: done

**Review by Sikora00 — APPROVED** [review]

**Pull request merged**